docs/modules/ROOT/pages/how-tos/release-process.adoc

```diff
@@ -25,10 +27,60 @@ git push -u
 
 . Check and update xref:tutorials/getting-started.adoc[Getting Started guide] and https://syn.tools/tuto/index.html[Tutorial: Writing your First Commodore Component] (These guides need to be up-to-date and work 100% all the time)
 
-== Additional work for major releases
+=== Additional work for major releases
 
 If a "major" release is being done, the following additional work needs to be done:
 
 * Update the https://github.com/projectsyn/documentation/blob/master/docs/modules/ROOT/pages/roadmap.adoc[Roadmap]
 * Write a blogpost and spread it on social media
 * Close the associated GitHub project and internal Jira project (if applicable)
+
+== Commodore Components
+
+Commodore Components are to be tagged with SemVer with `v<major>.<minor>.<patch>[-rc<rc>]` pattern (optionally includes Release candidates tagging).
+
+.Version examples
+====
+* `v1.0.0`
+* `v1.2.3-rc1`
+====
+
+[NOTE]
+====
+Generally, using `v0.x` versions are discouraged.
+After the first initial implementation, users are requested to directly release `v1.0.0`.
+Use release candidates suffix (e.g. `-rc1`) if "beta" versions are really needed.
+====
+
+The changelog is generated via GitHub Actions and is based on labelling PullRequests.
+The following GitHub labels require certain SemVer version increments:
+
+|===
+| Label | Increment
+
+| `bug`, `fix`, `change`, `dependency`
+| Patch
```

Comment:
Some remarks here:
* IMO, `change` can require a minor version bump, depending on the exact contents of the PR
* `dependency` can be anything from patch up to major depending on the changes present in the dependency.

Reply:
Hm, I would argue that `dependency` is coming from software development, but we do have component with dependent components. However, if those are updated with a major version, causing also a breaking change in "this" component, then `breaking` is a much more appropriate label than `dependency`.
To me, the label `dependency` should only be used if there's a (someday automated) dependency update that doesn't cause changes otherwise (we see those in Helm charts a lot for example). If they do cause changes, other labels would be more appropriate IMO.

`change` is probably only needed when something on the structure of the repository itself has changed (e.g. updated CI/CD pipeline), those don't warrant a minor version bump, maybe not even a new release (they will be added to the changelog on the next merged PR though). The only reason that `change` might be used is for announcing deprecations, everything else is either `enhancement` or `fix`, or do you see other uses examples?

Reply:
> Hm, I would argue that `dependency` is coming from software development, but we do have component with dependent components. However, if those are updated with a major version, causing also a breaking change in "this" component, then `breaking` is a much more appropriate label than `dependency`.
> To me, the label `dependency` should only be used if there's a (someday automated) dependency update that doesn't cause changes otherwise (we see those in Helm charts a lot for example). If they do cause changes, other labels would be more appropriate IMO.

I understood label `dependency` as "this PR updates a dependency" which could be anything from a patch level to a major level update of the dependency.
 
> `change` is probably only needed when something on the structure of the repository itself has changed (e.g. updated CI/CD pipeline), those don't warrant a minor version bump, maybe not even a new release (they will be added to the changelog on the next merged PR though). I see the only reason where `change` might be used for minor bump is for announcing deprecations, everything else is either `enhancement` or `fix`, or do you see other uses examples?

I've used `change` for PRs which are neither a pure bugfix, nor adding a new feature, e.g. https://github.com/projectsyn/component-rancher-monitoring/pull/19 where we reimplemented functionality that previously existed.

Reply:
I've extracted the PR labeling issue into #143 . Let's continue the discussion there.